jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/sam/SamCommonTest.kt

```diff
@@ -193,7 +193,7 @@ Resources:
             val dir = file.containingDirectory.virtualFile
             val templateFile = SamCommon.getTemplateFromDirectory(dir)
             assertNotNull(templateFile)
-            val codeUris = SamCommon.getCodeUrisFromTemplate(projectRule.project, templateFile)
+            val codeUris = SamCommon.getCodeUrisFromTemplate(projectRule.project, templateFile!!)
```

Comment:
If you use `import kotlin.test.assertNotNull` instead of the JUnit one you get contract support which means you won't need the `!!`

Reply:
Ohhh TIL

Reply:
shud prolly do a ktlint rool around this.

Reply:
Ya we have it in the backlog to ban Junit Assert already